src/static/css/content.less

```diff
@@ -316,6 +322,19 @@ figure {
 // Initiate loop
 .color-swatch-loop( length(@colors) );
 
+.color-table {
+    width: 100%;
+    margin-bottom: 40px;
```

Comment:
Minor: Convert to ems and use a more standard value for us (i.e., a multiple of 15).